src/js/common/utils/options-for-select.js

```diff
@@ -359,6 +359,7 @@ const states = {
     { label: 'Ohio', value: 'OH' },
     { label: 'Oklahoma', value: 'OK' },
     { label: 'Oregon', value: 'OR' },
+    { label: 'Other', value: 'Other' },
```

Comment:
I think you'll want to do something similar to the schema change instead of this, since this will mean Other shows up in the state list for addresses and on edu.

You could create a birth place states list and use that in the BirthInformation component (I think you'll have to concat Other to this states list and then sort it).